crypto/src/hash/mod.rs

```diff
@@ -68,7 +68,7 @@ impl Hasher for Blake3_256 {
     fn merge_with_int(seed: Self::Digest, value: u64) -> Self::Digest {
         let mut data = [0; 64];
```

Comment:
Prefer `let mut data = [0; 40]`, then `data[32..].copy_from_slice` for both hash schemes.

Reply:
fixed!